src/RoboSaverVirtualModule.sol

```diff
@@ -144,19 +160,36 @@ contract RoboSaverVirtualModule {
                                      CONSTRUCTOR
     //////////////////////////////////////////////////////////////////////////*/
 
-    constructor(address _delayModule, address _rolesModule, address _keeper, uint256 _buffer) {
+    constructor(address _delayModule, address _rolesModule, address _keeper, uint256 _buffer, uint16 _slippage) {
         delayModule = IDelayModifier(_delayModule);
         rolesModule = IRolesModifier(_rolesModule);
         keeper = _keeper;
         buffer = _buffer;
+        slippage = _slippage;
 
         CARD = delayModule.avatar();
+        BPT_STEUR_EURE_POOL_ID = BPT_STEUR_EURE.getPoolId();
+
+        BPT_STEUR_EURE_ASSETS[0] = IAsset(address(STEUR));
```

Comment:
now that that the constructor grabs the pool id programatically, maybe it is a good opp to do the same here for the assets

i.e:
```
(tokens, ,) = BALANCER_VAULT.getPoolTokens(BPT_STEUR_EURE_POOL_ID)
```

in that way slowly the code moves into more programatic way to determine the assets of a pool, indexes, its pool id etc. wdyt?

Reply:
i like it. maybe not necessary for the alpha version, but will probably be needed at some point

28ba46048f9fcafbafeb45d7a00dddcd9a2ded12

Reply:
nvm, i read it as you were not implementing it now 😆